tests/ddl_no_csc2.test/t00_create_table.expected

```diff
@@ -119,7 +119,7 @@ keys
 		uniqnulls "COMDB2_PK" = i 
 	}
 ')
-(type='index', name='$COMDB2_PK_5E8AC7D0', tbl_name='t1', rootpage=5, sql='create index "$COMDB2_PK_5E8AC7D0" on "t1" ("i");', csc2=NULL)
+(type='index', name='$COMDB2_PK_877B2989', tbl_name='t1', rootpage=5, sql='create index "$COMDB2_PK_877B2989" on "t1" ("i");', csc2=NULL)
```

Comment:
I wasn't expecting this to change. Do you know what could have caused this?

Reply:
The UNIQNULLS flag was not being handled when building the string to perform the CRC32 on.